plugins/opapolicyengine/go.mod

```diff
@@ -6,8 +6,8 @@ replace github.com/veraison/common => ../../common/
 
 require (
 	github.com/hashicorp/go-plugin v1.3.0
-	github.com/open-policy-agent/opa v0.40.0
-	github.com/stretchr/testify v1.7.1
+	github.com/open-policy-agent/opa v0.42.0
+	github.com/stretchr/testify v1.8.0
```

Comment:
happy with this too.  but just out of curiosity: why do we need this here?

Reply:
When I ran go mod tidy this updates automatically.